rest/cache.py

```diff
@@ -69,7 +70,7 @@ def get_int_setting(param, default):
     enabled = False
 
 
-def cache():
+def cache(userSpecific=False):
```

Comment:
Clearly, no. This cache function is not about jobs so there is no reason to filter jobs here. Caching and filtering are 2 different purposes that must be treated separately.